### ISSUE TITLE
Enhancement for multisignatures wallet contract

### ISSUE THREAD

**Issue by olivierstern** [ethereum/meteor-dapp-wallet#178]
Would be great to add a dropdown in the multisig wallet contract creation form for the account owner.
Otherwise the user needs to close the form copy and paste the adresses in a blank document, then startover the wallet contract creation...

**Comment by luclu** [comment]
Also this sould be availabe:

<img width="707" alt="bildschirmfoto 2016-05-26 um 17 01 05" src="https://cloud.githubusercontent.com/assets/10135872/15579117/7f19f3b4-2363-11e6-9094-88fe0b754662.png">

Might be related to https://github.com/ethereum/meteor-dapp-wallet/issues/176 ?!

**Comment by olivierstern** [comment]
No I meant here :
![select](https://cloud.githubusercontent.com/assets/19267090/15580866/7f418332-236a-11e6-82c9-a40613d76c9b.jpg)

**Comment by olivierstern** [comment]
There should be a possibility to paste or select from a list, no ?

**Comment by luclu** [comment]
Yes good point. This will be added during the addition of the addressbook-functionality.

**Comment by olivierstern** [comment]
:) Great !
